### ISSUE TITLE
Database Migration

### ISSUE THREAD

**Issue by on3ss** [LauraBeatris/goeat-api#10]
I've been trying to migrate the tables into the database (PostgreSQL) right after cloning and `yarn install` using both typeorm and sequelize however when migrating with typeorm i get this
`error Command "typeorm" not found.`
and with sequelize using this command 
`npx sequelize db:migrate`
i get this
```
Sequelize CLI [Node: 14.15.0, CLI: 5.5.1, ORM: 5.21.2]

Loaded configuration file "src\config\database.js".
```
After i installed typeorm and created the ormconfig.json as follows
```
{
    "type": "postgres",
    "host": "localhost",
    "port": 5432,
    "username": "postgres",
    "password": "",
    "database": "goeat"
 }
```

and rerun `yarn typeorm migration:run` with this response
```
yarn run v1.22.10
$ D:\Oness\code\PersonalProjects\goeat-api\node_modules\.bin\typeorm migration:run
Done in 3.44s.
```
but when i check my db, no tables were created

EDIT: 
```
package version = 1.0.0
nodejs version = v14.15.0
```

**Comment by LauraBeatris** [comment]
Hello @on3ss 

Thanks for creating this issue! 
My apologies, the reason for the error `error Command "typeorm" not found.` is that `typeorm` is not being used in this application... I've used it often in other projects and accidentally paste it here while I was updating the docs. 

The migrations should be run with `sequelize-cli`

I've fixed the migrations intruction at b2e7dc1625966e8fbb382352c7c794e3f96c472e

**Comment by LauraBeatris** [comment]
I'll close it since the issue was with the docs and I've already fixed at b2e7dc1 but let me know if you have any other problem to run the migrations
